### DISCUSSION TITLE
ColorSwatch does not display.

### DISCUSSION THREAD

**Discussion by YukiWorks432** [adobe/react-spectrum#7336] [Q&A]
ColorSwatch does not display well.
It does not display by itself like `<ColorSwatch color="#f00" />`.
It does not show up when put in a ColorSwatchPicker.
It may be displayed when it is put in a Grid, though,
If put it in a hierarchy like <Grid><ColorSwatchPicker><ColorSwatch>, it will not be displayed.
ColorSwatch put in a Grid seems to ignore rounded corners.

Created a skeleton with Bolt-CEP,
yarn add @adobe/react-spectrum
yarn add react-aria-components
I tested in an environment where we did the above two things.

OS: Windows11 Software used for testing: After Effects 2024

**Comment by snowystinger** [comment]
Looks like it's working https://react-spectrum.adobe.com/react-spectrum/ColorSwatch.html#colorswatch
Can you create a reproduction in codesandbox or stackblitz?

**Reply by YukiWorks432** [reply]
Thank you.
I tried with codesandbox and ColorSwatch displayed fine.
However, it did not display in the CEP extension where I copy-pasted the code.
Maybe there is a problem with the code build, etc.
I am not familiar with TypeScript, so I don't know what the cause is.

codesandbox link: https://codesandbox.io/p/sandbox/colorswatch-test-g65yws

**Reply by snowystinger** [reply]
I think the sandbox might be private or wasn't saved? I can't see it.

CEP is a different rendering engine, you may need to apply extra styles or debug in that environment to see what is happening.
